cloudformation/memsub-promotions-cf.yaml

```diff
@@ -57,6 +57,12 @@ Parameters:
     Description: Security group that grants access to the account's Vulnerability
       Scanner
     Type: AWS::EC2::SecurityGroup::Id
+  ReadableS3Resources:
+    Description: List of S3 resources that this app should be allowed to read
+    Type: CommaDelimitedList
+    AllowedValues:
```

Comment:
Unfortunately you can't use other parameters at this point, so we have to have two lists (PROD and CODE)

Reply:
Good idea, more lines but easier to read!